Selection_Sort_C/main.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+
+void PrintArray(int array[], int sizeOfArray)
+{
+    printf("Sorted array: \n");
```

Comment:
You don't know here if it is sorted or not. This method just print the array. Please remove the word `Sorted` from `printf`.